Resources/Locale/en-US/accent/italian.ftl

```diff
@@ -70,28 +74,33 @@ accent-italian-words-21 = nuke
 accent-italian-words-replace-21 = spiciest-a meatball
 
 accent-italian-words-22 = op
+accent-italian-words-22-2 = operative
 accent-italian-words-replace-22 = greek
 
-accent-italian-words-23 = operative
-accent-italian-words-replace-23 = greek
+accent-italian-words-23 = operatives
+accent-italian-words-23-2 = ops
+accent-italian-words-replace-23 = greeks
 
-accent-italian-words-24 = operatives
-accent-italian-words-replace-24 = greeks
```

Comment:
I don't really understand this. Perhaps this should be removed?

Reply:
I didnt add any new ones to the word list that is in the game currently, 

I just made sure that things that looked like this: 

accent-italian-words-22 = op
accent-italian-words-replace-22 = greek
accent-italian-words-23 = operative
accent-italian-words-replace-23 = greek

Was consolidated into:

accent-italian-words-22 = op
accent-italian-words-22-2 = operative
accent-italian-words-replace-22 = greek

Reply:
I realise now that it might have been better to split it into a separate pr just for that so that it didnt get jumbled up like this